docs/BUILD.dpdkstat.md

```diff
@@ -217,7 +217,7 @@ See also: http://dpdk.org/doc/guides/prog_guide/multi_proc_support.html
 
  *  The same PCI device configuration should be passed to the primary process as
     the secondary process uses the same port indexes as the primary.
- *  A blacklist / whitelist of NICs isn't supported yet.
+ *  A deny-list / allow-list of NICs isn't supported yet.
```

Comment:
I believe the prevalent terminology is `s/blacklist/blocklist/`, and `s/whitelist/allowlist/` (no dashes in "allowlist") — see, e.g., [this article](https://www.theregister.com/2019/09/03/chromium_microsoft_offensive/).

Reply:
Thank you for the quick review, I'll update the PR